.claude/commands/list-sessions.md

```
@@ -0,0 +1,133 @@
+---
+model: claude-haiku-4-5-20251001
+---
+
+# List Sessions
+
+Browse and search through saved conversation sessions.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)